sql/custom/world/world.DKS.sql

```diff
@@ -0,0 +1,4 @@
+-- npc id lich king
+-- Original npcflag was 2
+-- no previous scriptname
+UPDATE `creature_template` SET `npcflag`='3, `ScriptName`='npc_tc_skip_lich' WHERE  `entry`=25462;
```

Comment:
missing newline.

Reply:
> missing newline.

New line? im not understanding.
what this does is makes the first lich king npc u see have a additional  gossip for the option al skip.
![image](https://user-images.githubusercontent.com/16887899/126684607-48df78bd-e48f-4a24-ad41-3f704a749c38.png)

Reply:
> missing newline.

I believe i was informed by what you ment. I went ahead and added a new line. So we wait for travis to finish now. @Aokromes